bin/brew

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
The `[[...]]` lets you use operators like `==`, `<`, and so on, which can be more readable than `-gt`, `-le`, and so on. It also gives you the `=~` pattern matching operator.

In `zsh`, `[[...]]` is syntactically safer than `[...]`, which must have the stuff inside it expanded as though it were for an external command, so it can avoid some gotchas with special characters and the like. I think the same is true for bash: in `[[...]]`, word splitting and file expansion are not done, which is probably what you want, and can save some ugly quoting and/or escaping.